File: moneta/moneta/tag_example.py
import moneta_tags as moneta
import random


#moneta.DUMP_START_ALL("init", True);
t = [random.random() for _ in range(0,10000)]
#moneta.DUMP_STOP("init")
moneta.START_TRACE()
moneta.DUMP_START_ALL("sort", True);
s = t.sort()
moneta.DUMP_STOP("sort")

moneta.DUMP_START_ALL("shuffle", True);
s = random.shuffle(t)
moneta.DUMP_STOP("shuffle")

File: moneta/moneta/tag_example.js
const moneta = require( '../build/release/moneta.node');

t = Array.from({length: 1000}, () => Math.floor(Math.random() * 1000));
moneta.JS_START_TRACE();
// moneta.JS_DUMP_START_ALL("sort", true);
s = t.sort();
moneta.JS_DUMP_STOP("sort");
moneta.JS_DUMP_STOP("sort");
moneta.JS_NEW_TRACE("shuffle");
moneta.JS_DUMP_START_ALL("shuffle", true);
s = t.sort(()=> Math.random() - 0.5);
moneta.JS_DUMP_STOP("shuffle");